packages/js/product-editor/src/components/details-categories-field/create-category-modal.tsx

```diff
@@ -91,7 +92,7 @@ export const CreateCategoryModal: React.FC< CreateCategoryModalProps > = ( {
 					value={ categoryName }
 					onChange={ setCategoryName }
 				/>
-				<SelectControl< Pick< ProductCategory, 'id' | 'name' > >
+				<SelectControl< ProductCategoryLinkedList >
```

Comment:
This should probably also be replaced by the new TreeSelect, although that can be done in a follow up.